packages/sample-apps/codegen/NativeMyJsiModuleSpec.g.h

```diff
@@ -0,0 +1,101 @@
+
+/*
+ * This file is auto-generated from a NativeModule spec file in js.
+ *
+ * This is a C++ Spec class that should be used with MakeTurboModuleProvider to register native modules
+ * in a way that also verifies at compile time that the native module matches the interface required
+ * by the TurboModule JS spec.
+ */
+#pragma once
```

Comment:
>pragma [](http://example.com/codeflow?start=1&length=6)

This file does not look right yet. I guess it is still a TODO item.

Reply:
Whats wrong with it? -- This should be the NM2 spec for the same module.  (We happen to be using the c++ JSI spec to implement it in this case rather than this one)

Reply:
How does it work? I do not see generated code with the custom attributes. The spec cannot be verified if there are no custom attributes.